pypit/spectrographs/bpmimage.py

```diff
@@ -119,9 +119,9 @@ def build(self):
             if self.spectrograph in ['keck_lris_red']:
                 # Index in fitstbl for binning
                 xbin, ybin = [int(ii) for ii in self.binning.split(',')]
-                self.bpm = arlris.bpm(xbin, ybin, 'red', self.det)
+                self.bpm = lris.bpm(xbin, ybin, 'red', self.det)
```

Comment:
I think that the BPM stuff should all be included in the spectro_class. We shouldn't need the if/elif statements here.

Reply:
agreed.  Fixing.